backend/db/config.py

```diff
@@ -75,3 +75,52 @@ def get_pg_connect_url(local=False):
 if CONFIG["server"] == 'mysql':
     BRAND_NEW_DB_URL = BRAND_NEW_DB_URL + '?charset=utf8mb4'
 DB_URL = BRAND_NEW_DB_URL.replace(f'{CONFIG["port"]}', f'{CONFIG["port"]}/{CONFIG["db"]}')
+# DERIVED_TABLE_DEPENDENCIES: Shows which tables are needed to create a derived table. Generally the idea is that when
+# the dependency tables are updated, the dependent table also needs to be updated. But some tables in here have
+# dependencies but do not meet that use case. for example, 'concept_set_members_with_dups' depends on
+# 'concept_set_members', but that table is temporary.
+# todo: are the 'module names' part of the DDL file names the same as the keys in this dictionary? If so, write that
+#  down here as a comment. Could be useful for dynamically selecting the DDL files.
+DERIVED_TABLE_DEPENDENCIES = {
```

Comment:
### Recursively determining which derived tables need updating
@Sigfried I think this is the best way to do updating of derived tables. Basically, this is a list of tables, and the tables they depend on. So now that we have multiple kinds of refreshes, it will be useful to list out the tables that were just updated (e.g. 4 tables in the case of vocab), and then from `DERIVED_TABLE_DEPENDENCIES` we can recursively find all of the dependent derived tables, so we know exactly what to update.

This in theory is better than hard-coding the derived tables that need updating in the correct order, which is what I've done so far with the refresh.

I think this will make this PR take a little longer though.

Reply:
@Sigfried Just wanted to ask you about `omopconceptset` and `omopconceptsetcontainer`. Are we still using these tables? How are they created? From the objects API? What are they used for as opposed to `code_sets` and `concept_set_container`?